utils/users.js

```diff
@@ -196,6 +196,25 @@ const getRoleToUpdate = async (userData, newRoles) => {
   return { updateRole: true, newUserRoles };
 };
 
+const parseSearchQuery = (queryString) => {
+  const searchParams = {};
+  const queryParts = queryString.split(" ");
```

Comment:
There is already a function in helpers that splits query string and returns an object. Why are we not re-using that?

Reply:
Which function?

Reply:
Okay, got it. However, in this function, I modify the query value by converting it to an integer and lowercase, so we cannot use the getQualifiers function.